.github/workflows/LighthouseAudit/action.yaml

```diff
@@ -0,0 +1,81 @@
+name: LighthouseAudit
+description: Audit a URL with Lighthouse
+
+inputs:
+  url:
+    required: true
+    description: The url to audit
+  performance_threshold:
+    required: false
+    description: The performance threshold
+    default: '0.8'
+  accessibility_threshold:
+    required: false
+    description: The accessibility threshold
+    default: '0.8'
+  practices_threshold: 
+    required: false
+    description: The best-practices threshold
+    default: '0.7'
+  seo_threshold: 
+    required: false
+    description: The seo threshold
+    default: '0.8'
+  pwa_threshold:
+    required: false
+    description: The pwa threshold
+    default: '0'
+
+runs:
+  using: composite
+  steps:
+    - name: Perform Lighthouse Audit
```

Comment:
Failing the workflow is not yet added to this action, and it is stored in a json file, hence the steps before checking score thresholds. I had to do some work to get the data in the correct format.